src/client/entity-editor/submission-section/actions.ts

```diff
@@ -271,3 +259,33 @@ export function submit(
 			);
 	};
 }
+
+export function submitSingleEntity(submissionUrl:string, entityType:EntityTypeString, addEntity:(val)=>void, initialState = {}):SubmitResult {
+	return async (dispatch, getState) => {
+		const rootState = getState();
+		dispatch(setSubmitted(true));
+		const JSONState = rootState.toJS();
+		const entity = {...JSONState, type: entityType};
+		const postData = {
+			0: entity
+		};
+		try {
+			const response = await request.post(submissionUrl).send(postData);
+			const mainEntity = response.body[0];
+			const entityObject = {...initialState,
+				__isNew__: true,
```

Comment:
Yeah, needed only for knowing which entity is being created (edition, eg) and which are being edited (works..).
since entity editor using different routes for both operation, it doesn't need one.